crates/bevy_ui/src/render/mod.rs

```diff
@@ -366,8 +358,9 @@ pub fn extract_uinode_images(
 
         // Both vertical and horizontal percentage border values are calculated based on the width of the parent node
         // <https://developer.mozilla.org/en-US/docs/Web/CSS/border-width>
-        let parent_width = parent
-            .and_then(|parent| node_query.get(parent.get()).ok())
+        let parent_width = ui_children
```

Comment:
These border calculations are all getting moved out of extraction, afterwards there shouldn't be any need for any changes to the rendering module.

Reply:
Makes sense, in #15163 right?

I’ll try to do some reviews on that one as well as #9889 since both of these will require changes to this PR!

Reply:
Yep that's right.